### ISSUE TITLE
Extension formatter applies incorrect format to Svelte files

### ISSUE THREAD

**Issue by noahtallen** [biomejs/biome-zed#64]
Hi, I have a bug where the Biome extension is applying a different format within the editor as compared to the CLI.

Given some simple Svelte code, format on save in the IDE inserts indentation within the script tags, which is not expected from the Biome docs (https://biomejs.dev/internals/language-support/#html-super-languages-support)

```diff
// +layout.svelte
<script lang="ts">
+	const { children } = $props();
- const { children } = $props();
</script>
```

I'd expect it to not adjust that on save. In the CLI (`pnpm biome check`), it's not reported as an issue.

Zed settings:
```json
{
  "lsp": {
    "biome": {
      "settings": {
        "require_config_file": true
      }
    }
  },
  "code_actions_on_format": {
    "source.fixAll.biome": true,
    "source.organizeImports.biome": true
  }
}
```

Biome settings:
```json
{
	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
	"vcs": {
		"enabled": true,
		"clientKind": "git",
		"useIgnoreFile": true,
		"defaultBranch": "main"
	},
	"files": {
		"ignoreUnknown": false,
		"ignore": []
	},
	"formatter": {
		"enabled": true,
		"indentStyle": "tab"
	},
	"organizeImports": {
		"enabled": true
	},
	"linter": {
		"enabled": true,
		"rules": {
			"recommended": true
		}
	},
	"javascript": {
		"formatter": {
			"quoteStyle": "double"
		}
	}
}
```

**Comment by ematipico** [comment]
This is expected, and it's not related to the zed extension:

https://biomejs.dev/internals/language-support/#html-super-languages-support

**Comment by noahtallen** [comment]
After some more playing around, I was missing this bit of configuration in Zed, and now it's working. Oops!

```json
	"formatter": {
		"language_server": {
			"name": "biome"
		}
	},
```
